client/login/controller.js

```diff
@@ -139,3 +139,13 @@ export function redirectDefaultLocale( context, next ) {
 		context.redirect( '/log-in' );
 	}
 }
+
+export function redirectJetpack( context, next ) {
+	const { isJetpack } = context.params;
+	const { redirect_to } = context.query;
+
+	if ( ! isJetpack === 'jetpack' && includes( redirect_to, 'jetpack/connect' ) ) {
```

Comment:
😂 I couldn't get this working. Can you spot the problematic logic? 39c16b3491b678046ca7918ef4662e55b9e33a58

Reply:
I was just looking at that, lol